roles/ovirt-datacenter-cleanup/defaults/main.yml

```diff
@@ -1 +1,4 @@
 format_storages: false
+hosted_engine: false
```

Comment:
maybe we can remove it and set it dynamically ... 
 as I explained in roles/ovirt-datacenter-cleanup/README.md